makepkg/PKGBUILD

```diff
@@ -10,13 +10,11 @@ source=('git+https://github.com/AladW/aurutils')
 sha256sums=('SKIP')
 conflicts=('aurutils')
 provides=('aurutils')
-depends=('pacman>=5.0' 'git' 'jq' 'pacutils' 'wget')
+depends=('pacman>=5.0' 'git' 'jq' 'pacutils' 'wget' 'parallel')
```

Comment:
A possible place to add the `will cite` notice could be an `.install` file. That or a forum post when 2.0.0 is announced. It seems out of scope to be include this in `aur(1)` or a different manual.

Reply:
Are you referring something like just a warning or actually doing 
`printf 'will cite\n' | parallel --citation` as part of the `.install`?

I am not a lawyer, but:
Based on how the aur works, the commands in `.install` are technically run by the user, so I guess it could be an option. Especially if we use a custom `$PARALLEL_HOME` for aurutils, in order to isolate aurutils use of the tool.

Reply:
In this it would be a warning. If we wanted to run this automatically, it should be part of the prepare() function or such (since it needs to be run as the user running the PKGBUILD), though I guess this entails an unattended modification of the home directory and would thus be frowned upon.

I like the idea of using a custom `$PARALLEL_HOME`. Perhaps we could add a small wrapper to `lib` that sets it both to `~/.config/aurutils/parallel` and wraps any `parallel` commands with `--will-cite` (leaving a comment as to the licensing problematic)